.gitignore

```diff
@@ -1,38 +1,27 @@
-# Build files
-pom.xml
-*jar
-/lib/
-/classes/
+
+# Leiningen
 /target/
-/docs/
-/pkg/
-/log/
-*.tar.gz
-version
-junit
-
-# Emacs files
+
+# Emacs
 *#
+*~
+.#*
 
 # Local overide for testing config
-test-resources/config/local.clj
-
-# Vagrant cache folder & vagrantfile if used
-.vagrant
-Vagrantfile
```

Comment:
Just didn't realize we needed it (as a puppetdb-specific thing), and if we do, I wonder if perhaps it should be better constrained, i.e. maybe we'd want `git status` to tell us if there are say 5 vagrant dirs in src/puppetlabs/puppetdb.